AppSettingsManagerAPI/AppSettingsManagerApi.Infrastructure/Migrations/20230206201303_InitialCreate.cs

```diff
@@ -21,7 +21,11 @@ protected override void Up(MigrationBuilder migrationBuilder)
                                 .Annotation("MySql:CharSet", "utf8mb4"),
                             Password = table
                                 .Column<string>(type: "longtext", nullable: false)
+                                .Annotation("MySql:CharSet", "utf8mb4"),
+                            Email = table
+                                .Column<string>(type: "varchar(50)", maxLength: 50, nullable: false)
```

Comment:
In the future. Do not undo and redo migrations, just make a new one. Once we actually have the database deployed you won't be able to do this. In fact getting our migration history out of sync with the database would be a big problem

Reply:
@n898c were you able to add the Settings objects back in and test that those methods worked?

Reply:
Sorry. When you said update the migrations I thought you meant to just make changes to the existing ones. Would you like me to add a new migration? I can do that it's no problem. I would just need your guidance on how to do that.

Also, what do you mean by adding the Settings objects back? Like test these out?
![image](https://user-images.githubusercontent.com/74080527/220145627-00a6209c-387f-4050-8de3-7f8af27f0586.png)

Reply:
When I create a new user, should it also create a new entry in the settings? Because it isn't doing that. 
![image](https://user-images.githubusercontent.com/74080527/220148010-6d0cc15a-7479-407b-b4ed-0b8460bb9fa5.png)
![image](https://user-images.githubusercontent.com/74080527/220148139-1eea3679-9101-4b69-b598-a8f04ee7013f.png)

Reply:
No we don't have them actually linked yet. The methods are all separate. Should be able to test the operations just on setting objects.

Doing a new migration is pretty much the same as what I think you did which is run "dotnet ef migrations add <migration_name>" right?

Reply:
If this works then it's fine, don't worry too much about it